lib/index.js

```diff
@@ -23,7 +50,8 @@ module.exports = {
 
     const serverOptions = {
       name: config.get('name'),
-      version: packageJson.version
+      version: packageJson.version,
+      handleUncaughtExceptions: true  //deprecated, but without an equivalent alternative as of restify 8.5.1
```

Comment:
Will need to watch this, looks like it may be out when node domains is removed (probably around v14).